aspnetcore/choose-aspnet-framework.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Choose between ASP.NET and ASP.NET Core | Microsoft Docs"
+title: "Choose between ASP.NET and ASP.NET Core "
```

Comment:
You're leaving trailing spaces - can you revert and redo with the initial space in the search string?